app/helpers/common_parameters_helper.rb

```diff
@@ -40,7 +40,8 @@ def use_puppet_default_help link_title = nil, title = _("Use Puppet default")
   end
 
   def hidden_value_field(f, field, value, disabled, options = {})
-    if f.object.hidden_value?
+    hidden = options.delete(:hidden_value)
+    if hidden || f.object.hidden_value?
```

Comment:
`f.object.hidden_value?` will have the same value as `hidden` or it will be nil so if `hidden` is false the whole condition will be false anyway so that's fine.